src/data/faq_de.json

```diff
@@ -871,7 +871,7 @@
+                                    "<b>Aktualisiert am 13. Oktober 2022:</b><div class='container'><div class='row'><div class='col-sm'><a href='/assets/img/faq/vac_booster_basics_de.png' target='_blank'><img class='img-fluid' src='/assets/img/faq/vac_booster_basics_de.png' alt='Zertifikatenliste' width='300'></a></div><div class='col-sm'><p>Die Auffrischimpfung (Boosterimpfung) ist ein Impfvorgang, der nach der Grundimmunisierung den nachlassenden Impfschutz auffrischen soll. In den meisten Fällen, aber nicht immer, findet Sie gemäß den Empfehlungen der Behörden nach einer Zeit von 6 Monaten (zuvor 3 Monate) nach dem letzten Impfvorgang statt. Typisches Beispiel: Erfolgte der Abschluss der Impfreihe zur Grundimmunisierung am 08.10.2022, dann könnte eine Auffrischimpfung am 08.04.2023 erfolgen.</p><p>Unter <a href='https://www.zusammengegencorona.de/impfen/#id-862101356'>Impf Guide</a> vom Bundesministerium für Gesundheit (BMG) finden Sie einen \"Impf-Guide\" zu Ihrer persönlichen Impfempfehlung von der Ständigen Impfkommission (STIKO)</p><p>Jede Impfung kann durch ein digitales COVID-Zertifikat dokumentiert werden, das Sie in die Corona-Warn-App einlesen können. <b>Wichtig</b>: Bitte achten Sie schon direkt beim Ausstellen des Zertifikats darauf, dass die persönlichen Daten (Titel, Name, Vornamen, und Geburtsdatum) passend zu den bereits existierenden Impfzertifikaten (und auch ggfs. Genesenenzertifikaten) ausgewiesen werden. Nur so kann die Corona-Warn-App das neue Zertifikat zu den anderen Zertifikaten (Impfungen bzw. Genesung) zuordnen (Beispiel: gleicher akademischer Titel oder Angabe des vollständigen Doppelnamens). Alle Zertifikate einer Person sollten in der Corona-Warn-App gruppiert angezeigt werden, siehe Abbildung.</p><p>Auffrischimpfungen bieten einen sofortigen Impfschutz.</p><p>Basierend auf den bisherigen Impfvorgängen, wird das erste Zertifikat einer Auffrischimpfung mit '4 von 4' (bei vorheriger Grundimmunisierung mit 3 Dosen) ausgestellt.<p><p>Die Kennzeichnung '2 von 1' im Zertifikat einer Impfung zeigt an, dass es sich um den ersten Impfvorgang nach einer Genesung handelt.</p><p>Siehe FAQ-Artikel: <a href='#vac_booster_jj' target='_blank'>Warum erscheint in der Corona-Warn-App eine 14-tägige Wartezeit bis zum vollen Impfschutz nach einer Auffrischimpfung?</a></p></div></div></div>"

```

Comment:
For the new paragraph suggest:

<p>Unter <a href='https://www.zusammengegencorona.de/impfen/#id-862101356'>Impf-Guide</a> vom Bundesministerium für Gesundheit (BMG) können Sie Ihre persönlichen Impfempfehlung von der Ständigen Impfkommission (STIKO) erstellen.</p>

Reply:
@Ein-Tim 
Thanks! I changed the suggestion.

Reply:
Thank you both, included it!